word2vec_decade_experiment/02_word2vec_decade_analysis.py

```diff
@@ -16,58 +16,147 @@
 # # Analyze Word2Vec by Decades Run
```

Comment:
This is something a bit personal: I put a brief description of the notebook at the top.